src/data/polynomial/monic.lean

```diff
@@ -131,6 +135,84 @@ by rw [eq_C_of_degree_le_zero this, ← nat_degree_eq_zero_iff_degree_le_zero.2
 
 end comm_semiring
 
+section comm_ring
+variables [comm_ring R]
+namespace monic
+
+lemma coeff_nat_degree {p : polynomial R} (hp : p.monic) : p.coeff (p.nat_degree) = 1 := hp
+
+@[simp]
+lemma degree_one {p : polynomial R} (hp : p.monic) :
+p.nat_degree = 0 ↔ p = 1 :=
```

Comment:
I don't think this name is clear. Since it's a `simp` lemma, there's little harm in being verbose. Perhaps just `degree_eq_zero_iff_eq_one`? (Since it's under the `monic` namespace that's then clear.)